e2e/testcases/helm_sync_test.go

```diff
@@ -289,6 +640,121 @@ func TestHelmNamespaceRepo(t *testing.T) {
 	}
 }
 
+// TestHelmNamespaceRepoWithConfigMap verifies RepoSync can pick up values and updates from ConfigMap references.
+// This test will work only with following pre-requisites:
+// Google service account `e2e-test-ar-reader@${GCP_PROJECT}.iam.gserviceaccount.com` is created with `roles/artifactregistry.reader` for accessing images in Artifact Registry.
+func TestHelmConfigMapNamespaceRepo(t *testing.T) {
+	repoSyncNN := nomostest.RepoSyncNN(testNs, configsync.RepoSyncName)
+	nt := nomostest.New(t, nomostesting.SyncSource, ntopts.RequireGKE(t),
+		ntopts.RepoSyncPermissions(policy.RepoSyncAdmin(), policy.AppsAdmin(), policy.CoreAdmin()),
```

Comment:
Why does the ns reconciler need `RepoSyncAdmin`?

Reply:
You're right, it doesn't. Took it out, thanks for the catch!